src/components/combobox/combobox.e2e.ts

```diff
@@ -481,21 +471,188 @@ describe("calcite-combobox", () => {
+
+      await page.keyboard.press("ArrowDown");
+      await page.waitForChanges();
+      let popper = await page.find("#parentOne >>> .popper-container--active");
+      expect(popper).toBeTruthy();
+
+      await page.keyboard.press("Tab");
+      await page.waitForChanges();
+      popper = await page.find("#parentOne >>> .popper-container--active");
+      expect(popper).toBeNull();
+    });
+
+    it(`ArrowDown opens the item group for combobox in focus and jumps to the first item`, async () => {
+      const inputEl = await page.find(`#parentOne >>> input`);
+      await inputEl.focus();
+      await page.waitForChanges();
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+
+      await page.keyboard.press("ArrowDown");
+      await page.waitForChanges();
+      const firstFocusedGroupItem = await page.find("#one >>> .label--active");
+      expect(firstFocusedGroupItem).toBeTruthy();
+    });
+
+    it(`Escape closes the dropdown, but remains focused`, async () => {
+      const inputEl = await page.find(`#parentOne >>> input`);
+      await inputEl.focus();
+      await page.waitForChanges();
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+
+      await page.keyboard.press("ArrowDown");
+      await page.waitForChanges();
+      let popper = await page.find("#parentOne >>> .popper-container--active");
+      expect(popper).toBeTruthy();
 
+      await page.keyboard.press("Escape");
+      await page.waitForChanges();
+      popper = await page.find("#parentOne >>> .popper-container--active");
+      expect(popper).toBeNull();
+
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+    });
+
+    it(`home opens dropdown and puts focus on first item`, async () => {
+      const inputEl = await page.find(`#parentOne >>> input`);
+      await inputEl.focus();
+      await page.waitForChanges();
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+
+      await page.keyboard.press("Home");
+      await page.waitForChanges();
+      const firstFocusedGroupItem = await page.find("#one >>> .label--active");
+      expect(firstFocusedGroupItem).toBeTruthy();
+
+      const visible = await firstFocusedGroupItem.isVisible();
+      expect(visible).toBe(true);
+    });
+
+    it(`end opens dropdown and puts focus on last item`, async () => {
+      const inputEl = await page.find(`#parentOne >>> input`);
+      await inputEl.focus();
+      await page.waitForChanges();
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+
+      await page.keyboard.press("End");
+      await page.waitForChanges();
+      const lastFocusedGroupItem = await page.find("#three >>> .label--active");
+      expect(lastFocusedGroupItem).toBeTruthy();
+
+      // follow up issue on existing bug: the last item in focus won't scroll into view.
```

Comment:
Can we link the issue here? or maybe we can get it working with some kind of timeout call?